packages/SwingSet/src/index.js

```diff
@@ -12,3 +12,6 @@ export { buildTimer } from './devices/timer.js';
 export { buildBridge } from './devices/bridge.js';
 export { default as buildCommand } from './devices/command.js';
 export { buildPlugin } from './devices/plugin.js';
+
+/** @typedef { import('./types').BundleID } BundleID */
```

Comment:
I'm gonna be stubborn, but I'd still prefer using extensions.
```suggestion
/** @typedef { import('./types.js').BundleID } BundleID */
```

Reply:
Note that when VS Code offers to import, it does so without the `.js`, even with the `typescript.preferences.importModuleSpecifierEnding": "js"` setting.

Reply:
Interesting, I swear it does for me. We should find what setting triggers this.

Reply:
Oh good! We shouldn't have to decide based on limits of tooling. Once we have a setting we agree on we can distribute with https://github.com/Agoric/agoric-sdk/pull/4418

Reply:
Well apparently I must have muscle memory and add those automatically as I can't seem to reproduce anymore.

Reply:
Oh, I thought I got red squiggles from vscode when I did that. I prefer extensions too, I'll look into it.